MediaBrowser.WebDashboard/Api/PackageCreator.cs

```diff
@@ -1,161 +0,0 @@
-        /// </returns>
-        public static async Task<Stream> ModifyHtml(
-            bool isMainIndexPage,
-            Stream sourceStream,
-            string mode,
-            string appVersion,
-            string localizationCulture)
-        {
-            string html;
-            using (var reader = new StreamReader(sourceStream, Encoding.UTF8))
-            {
-                html = await reader.ReadToEndAsync().ConfigureAwait(false);
-            }
-
-            if (isMainIndexPage && !string.IsNullOrWhiteSpace(localizationCulture))
-            {
-                var lang = localizationCulture.Split('-')[0];
-
-                html = html.Replace("<html", "<html data-culture=\"" + localizationCulture + "\" lang=\"" + lang + "\"", StringComparison.Ordinal);
-            }
-
-            if (isMainIndexPage)
-            {
-                html = html.Replace("<head>", "<head>" + GetMetaTags(mode), StringComparison.Ordinal);
-            }
-
-            // Disable embedded scripts from plugins. We'll run them later once resources have loaded
-            if (html.IndexOf("<script", StringComparison.OrdinalIgnoreCase) != -1)
-            {
-                html = html.Replace("<script", "<!--<script", StringComparison.Ordinal);
-                html = html.Replace("</script>", "</script>-->", StringComparison.Ordinal);
-            }
-
-            if (isMainIndexPage)
-            {
-                html = html.Replace("</body>", GetCommonJavascript(mode, appVersion) + "</body>", StringComparison.Ordinal);
-            }
-
-            var bytes = Encoding.UTF8.GetBytes(html);
-
-            return new MemoryStream(bytes);
-        }
-
-        /// <summary>
-        /// Gets the meta tags.
-        /// </summary>
-        /// <returns>System.String.</returns>
-        private static string GetMetaTags(string mode)
-        {
-            var sb = new StringBuilder();
-
-            if (string.Equals(mode, "cordova", StringComparison.OrdinalIgnoreCase)
-                || string.Equals(mode, "android", StringComparison.OrdinalIgnoreCase))
-            {
-                sb.Append("<meta http-equiv=\"Content-Security-Policy\" content=\"default-src * 'self' 'unsafe-inline' 'unsafe-eval' data: gap: file: filesystem: ws: wss:;\">");
-            }
-
-            return sb.ToString();
-        }
-
-        /// <summary>
-        /// Gets the common javascript.
-        /// </summary>
-        /// <param name="mode">The mode.</param>
-        /// <param name="version">The version.</param>
-        /// <returns>System.String.</returns>
-        private static string GetCommonJavascript(string mode, string version)
-        {
-            var builder = new StringBuilder();
-
-            builder.Append("<script>");
-            if (!string.IsNullOrWhiteSpace(mode))
-            {
-                builder.AppendFormat(CultureInfo.InvariantCulture, "window.appMode='{0}';", mode);
```

Comment:
where is this handled/used?

Reply:
This is automatically set for the standalone flavor.